pytorch_lightning/core/saving.py

```diff
@@ -202,7 +202,13 @@ def _load_model_state(cls, checkpoint: Dict[str, Any], strict: bool = True, **cl
         model.on_load_checkpoint(checkpoint)
 
         # load the state_dict on the model automatically
-        model.load_state_dict(checkpoint['state_dict'], strict=strict)
+        keys = model.load_state_dict(checkpoint['state_dict'], strict=strict)
+
+        if not strict:
+            if keys.missing_keys:
+                rank_zero_warn(f"Found keys that are in model state dict but not in checkpoint: {keys.missing_keys}")
+            if keys.unexpected_keys:
+                rank_zero_warn(f"Found keys that are not in model state dict but in checkpoint: {keys.unexpected_keys}")
```

Comment:
I personally prefer the "the"s

```suggestion
                rank_zero_warn(f"Found keys that are in the model state dict but not in the checkpoint: {keys.missing_keys}")
            if keys.unexpected_keys:
                rank_zero_warn(f"Found keys that are not in the model state dict but in the checkpoint: {keys.unexpected_keys}")
```

Reply:
Should I send this change as a commit immediately?

Reply:
Sure why not :)

You can click the button "Commit suggestion" to push it directly

![Screenshot 2021-06-30 at 18 09 27](https://user-images.githubusercontent.com/11067892/123995157-55265d00-d9ce-11eb-8e20-d85d2ece9fe8.png)